### ISSUE TITLE
SnapshotRestoreException when attempting a restore from s3 repository

### ISSUE THREAD

**Issue by gregeinfrank** [elastic/elasticsearch#22973]
**Elasticsearch version**:
1.5

**Plugins installed**: [AWS Cloud plugin]

**JVM version**:
? (using aws elasticsearch service)
**OS version**:
? (using aws elasticsearch service)

**Description of the problem including expected versus actual behavior**:
{"error":"SnapshotRestoreException[[repository_name:snapshot-name] indices [my_index] and [my_index] are renamed into the same index [my_index]]","status":500}

**Steps to reproduce**:
 1. Start restore from a recent snapshot

**Provide logs (if relevant)**:

**Describe the feature**:

**Comment by dadoonet** [comment]
1.x series is now EOL'ed.
Can you upgrade to 2.4 or better to 5.2 and test again?

BTW you didn't provide the command you ran.

**Comment by gregeinfrank** [comment]
@dadoonet thanks for replying - and sorry for the false alarm here - I finally just figured out the problem. Documenting for anyone in the future that does the same thing. 

I had code that was generating the list of indices to restore, but did not realize that the list of indices contained duplicates for whatever reason.

```
POST /_snapshot/my_backup/snapshot_1/_restore
{
    "indices": "my_index,my_index", 
}
```

After I realized that the list of indices had duplicates, that error message `indices [my_index] and [my_index] are renamed into the same index [my_index]` made a lot more sense. 🤦‍♂️
